app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,116 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="app_name">MedTimer</string>
+  <string name="tab_settings">Instellingen</string>
+  <string name="tab_overview">Overzicht</string>
+  <string name="tab_medicine">Geneesmiddel</string>
+  <string name="add_medicine">Medicijn toevoegen</string>
+  <string name="medicine_name">Naam medicijn</string>
+  <string name="add_reminder">Herinnering toevoegen</string>
+  <string name="edit">Bewerking</string>
+  <string name="delete">Verwijderen</string>
+  <string name="time">Tijd</string>
+  <string name="dosage">Dosering</string>
+  <string name="yes">Ja</string>
+  <string name="ok">OK</string>
+  <string name="cancel">Annuleren</string>
+  <string name="confirm">Bevestigen</string>
+  <string name="are_you_sure_delete_medicine">Weet u zeker dat u het geneesmiddel wilt verwijderen?</string>
+  <string name="no_reminders">Geen herinneringen ingesteld</string>
+  <plurals name="som_herinneringen">
+       <item quantity="een">%1$d herinnering\n%2$s</item>
```

Comment:
The attribute may not be translated - this should stay "one" and "other" in line 22

Reply:
So you mean line 21 and 22 should not be translated ?

Reply:
Ok sorry it should be fixed now ;)